charts/monitoring-config/rules/search_api_v2_tests.yaml

```diff
@@ -43,12 +43,20 @@ tests:
               dataset_type: "invariants"
               severity: "warning"
               destination: "slack-search-quality-monitoring"
+            exp_annotations:
+              summary: Low score for invariant dataset dataset1
+              description: The score on invariant dataset dataset1 is too low (94%)
+              slack_bullet: '• *dataset1*: 94%'
           - exp_labels:
               alertname: LowInvariantScore
               dataset_name: "dataset5"
               dataset_type: "invariants"
               severity: "warning"
               destination: "slack-search-quality-monitoring"
+            exp_annotations:
+              summary: Low score for invariant dataset dataset5
+              description: The score on invariant dataset dataset5 is too low (94%)
+              slack_bullet: '• *dataset5*: 94%'
```

Comment:
Might make more sense to do the formatting in the message to slack, rather than worry about it in the annotation.